dnf-docker-test/features/steps/test_behave.py

```diff
@@ -64,6 +65,14 @@ def diff_package_lists(a, b):
         map(_right_decorator, list(sb - sa)))
 
 
+def diff_query_lists(sack_a, sack_b):
```

Comment:
saving `sack_a.query().installed()` didn't work? Do you need anything else from the sack like available packages etc.?